package.json

```diff
@@ -150,6 +150,7 @@
         "@govflanders/vl-ui-util": "14.0.2",
         "@ungap/custom-elements": "1.2.0",
         "bootstrap-sass": "3.4.3",
+        "construct-style-sheets-polyfill": "^3.1.0",
```

Comment:
we vermijden ~ of ^ zodat de versies zo vast als mogelijk liggen en we geen 'automatische' upgrades krijgen